File: api/src/API.Domain/RedeSocial.cs
namespace API.Domain
{
  public class RedeSocial
  {
    public int Id { get; set; }
    public string Nome { get; set; }
    public string URL { get; set; }
    public int? EventoId { get; set; }
    public EventosList Evento { get; set; }
    public int? AtracaoId { get; set; }
    public Atracao Atracao { get; set; }
  }
}

File: src/app/models/RedeSocial.ts
import { Atracao } from './Atracao';
import { Evento } from './Evento';

export interface RedeSocial {
  id: number;
  nome: string;
  URL: string;
  eventoId: number;
  evento: Evento[];
  atracaoId: number;
  atracao: Atracao;
}
